skeleton/template.yaml

```
@@ -45,8 +45,33 @@ spec:
           enum:
             - llama.cpp
             - vLLM
+            - Existing model server
           default: llama.cpp
           ui:help: If you choose vLLM, ensure your cluster has Nvidia GPU supported, and is with enough cpu and memory
+      dependencies:
+        modelServer:
+          oneOf:
+            - required:
+                - modelEndpoint
+                - modelName
+              properties:
+                modelServer:
+                    const: Existing model server
+                modelEndpoint:
+                  title: Model Server Endpoint
+                  type: string
+                  ui:help: "The endpoint for an existing model server."
```

```suggestion
                  description: "The endpoint for an existing model server."
```

same as other descriptions